.github/workflows/aikido-scan.yml

```
@@ -0,0 +1,69 @@
+  group: aikido-${{ github.workflow }}-${{ github.ref }}
+  cancel-in-progress: true
+
+permissions:
+  contents: read
+
+jobs:
+  aikido-scan:
+    name: Aikido Security Scan
+    runs-on: ubuntu-latest
+    timeout-minutes: 30
+    if: ${{ vars.AIKIDO_REPOSITORY_ID != '' && secrets.AIKIDO_CLIENT_API_KEY != '' }}
```

actionlint (1.7.12)

[error] context "secrets" is not allowed here. available contexts are "github", "inputs", "needs", "vars". see https://docs.github.com/en/actions/learn-github-actions/contexts#context-availability for more details

(expression)